main.js

```diff
@@ -1,6 +1,56 @@
-import './style.css'
+const key = "18acb5642976765d66ed9ea326327f5c";
+const BASEURL = "https://api.themoviedb.org/3";
```

Comment:
When using all caps for variable names in javascript, it is better to use kebab_case;
```suggestion
const BASE_URL = "https://api.themoviedb.org/3";
```